src/etc/inc/filter.inc

```diff
@@ -483,6 +483,23 @@ function filter_configure_sync($delete_states_if_needed = true) {
 		$_grbg = exec("/sbin/pfctl -t bogonsv6 -T kill 2>/dev/null");
 	}
 
+	// get aliases names from config
+	$aliasesnames = array();
+	foreach ($config['aliases']['alias'] as $alias) {
+		array_push($aliasesnames, $alias['name']);
+	}
+	// get aliases names from pfctl -sT
+	$pftables = explode("\n",shell_exec("/sbin/pfctl -sT"));
+	// compare config and pfctl aliases
+	$diff = array_diff($pftables, $aliasesnames);
+	// don't delete special aliases
+	$diff_exceptions = array('bogons','bogonsv6','snort2c','sshguard','negate_networks','tonatsubnets','virusprot','vpn_networks');
```

Comment:
We already have this list as a global `$reserved_table_names` in `/usr/local/www/guiconfig.inc` -- the list could be relocated to `/etc/inc/globals.inc` so it can be used here as well. No need to duplicate it.

Reply:
fixed